indico/modules/events/sessions/util.py

```diff
@@ -59,3 +69,71 @@ def can_manage_sessions(user, event, role=None):
     """Check whether a user can manage any sessions in an event"""
     return event.can_manage(user) or any(s.can_manage(user, role)
                                          for s in event.sessions.options(joinedload('acl_entries')))
+
+
+def generate_csv_from_sessions(sessions):
+    """Generate a CSV file from a given session list.
+
+    :param sessions: The list of sessions to include in the file
+    """
+    column_names = ['ID', 'Title', 'Description', 'Code']
+    buf = BytesIO()
+    writer = csv.DictWriter(buf, fieldnames=column_names)
+    writer.writeheader()
+    for sess in sessions:
+        row = {'ID': sess.friendly_id, 'Title': sess.title, 'Description': sess.description, 'Code': sess.code}
```

Comment:
AFAIK the CSV writer doesn't support unicode; you need to `.encode('utf-8')` the strings...